lego/tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+# Pytest configuration
```

Comment:
So there is a pytest.ini file for each test?

Reply:
No, for each test directory. One for the example and one for the lego test